README.md

```diff
@@ -1,159 +1,136 @@
+automatically copied together with the rest of the files to be served by the
+webserver. The *index.html* file (in the *public* folder of this repository)
+links already to *env-config.js*, which is read every time the user opens the
+application in a browser.
 
-The second step is to generate the definitions from the YAML translations and from OpenApi specs.
+>**IMPORTANT**: The *env.sh* script reads and automatically injects in
+>`env-config.js` all environment variables prefixed with *IO_ONBOARDING_PA*, for
+>example *IO_ONBOARDING_PA_API_HOST*.
 
-To generate the definitions from the YAML translations:
+To read the variable values inside the frontend application, use
+`window._env_.IO_ONBOARDING_PA_YOUR_VAR`. 
 
-```
-$ yarn generate:locales
-```
+## Production deployment
 
-To generate the definitions from the OpenApi specs:
+The frontend docker image is deployed in production on top of a Kubernetes
+cluster.
 
-```
-$ yarn generate:locales
-```
+For more informations about IO production deployments and helm-charts, see
+[io-infrastructure-post-config
+repository](https://github.com/teamdigitale/io-infrastructure-post-config).
 
-These commands can be run together using:
+Currently an Azure development pipeline is configured in order to deploy the
+master branch of this repository into the IO infrastructure staging environment;
+the process is triggered by every push into master.
 
-```
-$ yarn generate:all
-```
+The deployed frontend is reachable at
+[https://pa-onboarding.dev.io.italia.it/](https://pa-onboarding.dev.io.italia.it/).
 
 ## Test the application locally
 
-The application can be built and tested locally, either directly on the developer machine (directly using development tools, such as *yarn* or *parcel*), or as a Docker-based image.
-
-### Test locally with mocked APIs
-
-To test the frontend application locally, on developer machine, against some mocked APIs, run in the root folder
+Set up your environment variables first:
 
 ```shell
-yarn start-with-mock
+cp env.example .env
+# edit .env file
 ```
 
-Then, point your browser to
-
-* [http://localhost:1234](http://localhost:1234) for the frontend
-* [http://localhost:3000](http://localhost:3000) to reach the mocked backend
-
-The project uses the [json-server package](https://github.com/typicode/json-server) to mock the REST API.
-
-### Test with Docker
+### Develop the frontend
 
-To test the application, run in the root folder:
+To run the frontend application locally, prepare the environment first:
 
 ```shell
-# Copy the SPID demo configurations folder from the application backend repository
-git clone https://github.com/teamdigitale/io-onboarding-pa-api.git
-cp -r io-onboarding-pa-api/certs .
-cp -r io-onboarding-pa-api/testenv2 .
-rm -rf io-onboarding-pa-api
-
-# Bring up the Docker test environment (build the frontend, pull the backend and the SPID test environment)
-docker-compose up
+# run yarn inside docker container
```

Comment:
instead # Launch docker image and run yarn inside docker 

for me was not trivial to understand without going into the scripts